helpdesk_mgmt_ticket_closed_date/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,31 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, api, _, fields
+from odoo.exceptions import ValidationError
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    # default=True bug
+    closed_date_permits = fields.Boolean(
```

Comment:
Better naming suggestion:
```suggestion
    closed_date_editable = fields.Boolean(
```